filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -2455,10 +2454,10 @@ GLsizei OpenGLDriver::getAttachments(AttachmentArray& attachments,
         assert_invariant(!defaultFramebuffer);
         attachments[attachmentCount++] = GL_COLOR_ATTACHMENT7;
     }
-    if (any(buffers & TargetBufferFlags::DEPTH)) {
+    if (any(buffers & TargetBufferFlags::DEPTH) && rt->width != 0 && rt->height != 0) {
         attachments[attachmentCount++] = defaultFramebuffer ? GL_DEPTH : GL_DEPTH_ATTACHMENT;
     }
-    if (any(buffers & TargetBufferFlags::STENCIL)) {
+    if (any(buffers & TargetBufferFlags::STENCIL) && rt->width != 0 && rt->height != 0) {
```

Comment:
Did you verify that we still need this extra check? It was more of a shortcut solution, and we should understand if 1) it still causes an assert in ANGLE and 2) how framebuffers or attachments of zero dimensions may be born.